release notes/v0.45.0.md

```diff
@@ -0,0 +1,168 @@
+  });
+
+  // send a message to the server
+  stream.write({
+    lo: {
+      latitude: 400000000,
+      longitude: -750000000,
+    },
+    hi: {
+      latitude: 420000000,
+      longitude: -730000000,
+    },
+  });
+
+  sleep(0.5);
+};
+```
+</details>
+
+You can just replace `k6/grpc` import with `k6/experimental/grpc` to use the new functionality.
+
+In the future this functionality will be moved to the `k6/grpc` module.
+
+### You can now only upload a test to the cloud without running it [#3030](https://github.com/grafana/k6/pull/3030)
+
+For [years users have wanted](https://github.com/grafana/k6-cloud-feature-requests/issues/22) to be able to update the test that is saved in the cloud but *not* run it at this exact point.
+
+This is now possible by adding `--upload-only` when invoking `k6 cloud` as in `k6 cloud --upload-only script.js`.
+
+This is likely going to be most useful in a CI on the actual test script project. Now that CI can just run `k6 cloud --upload-only new-verson-of-script.js` on "release".
+
+And later on that newer version will be used. For example by a [scheduled run](https://k6.io/docs/cloud/manage/scheduled-tests/). 
+
+### Setting sample metadata API [#3037](https://github.com/grafana/k6/pull/3037)
+
+Support for high-cardinality metrics metadata was added in v0.41.0, but it wasn't accessible from test scripts. It's now possible to set or delete metadata for the whole VU with a similar API as used for tags:
+```javascript
+import exec from "k6/execution";
+
+export default () => {
+    exec.vu.metrics.metadata["my_cool_id"] = "a very unique value"
+    // all metrics from here on will have this metadata set
+    delete exec.vu.metrics.metadata["my_cool_id"]
+    // all metriccs from here on wil *not* have the metadata set
+}
+```
+
+This also introduces the sub-object `metrics` on the `vu` object.
+Apart from `metadata` it has another property `tags`. This is meant to be the new way to set tags instead of using `exec.vu.tags`. 
+
+There are no current plans to replace `exec.vu.tags` with `exec.vu.metrics.tags`.
+
+### UX improvements and enhancements
+
+- [#3015](https://github.com/grafana/k6/pull/3015) We're now building ARM64 Docker images in CI, but due to some Docker complications, they're still not pushed to the repository. We expect to resolve these issues by next release. Thanks @nrxr! 🙇
+- [#3099](https://github.com/grafana/k6/pull/3099) replace "breached" with "crossed" in logs around thresholds. Thanks to @MattDodsonEnglish 🙇.
+- [#3102](https://github.com/grafana/k6/pull/3102) Better error message when SharedArray constructor is provided with an async function. This is not supported, but the original message wasn't very clear.
+- [#3089](https://github.com/grafana/k6/pull/3089) Add Software Bill of Materials (SBOM) reports to k6 releases. Thanks to @SadFaceSmith 🙇.
+
+## Bug fixes
+
+* [#3058](https://github.com/grafana/k6/pull/3058) fix repetitive `the`. Thank you, @cuishuang 🙇.
```

Comment:
This is not a bug fix. Let's move it to the "Maintenance and internal improvements" section?